example/tactile/basic.py

```
@@ -0,0 +1,51 @@
+"""
+
+import time
+import numpy as np
+from wujihandpy import tactile
+
+
+def main():
+    with tactile.Glove() as glove:
+        info = glove.get_device_info()
+        build = glove.get_fw_build()
+        print(f"Tactile glove")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)